### ISSUE TITLE
Is anyone doing experiments only on mini-Imagenet? I find it hard to converge

### ISSUE THREAD

**Issue by mellthebs** [google-research/meta-dataset#67]
Is it because the pre-training model is too complex?

**Comment by mellthebs** [comment]
I found the training loss is very big

**Comment by foreversolar** [comment]
@mellthebs 你好，可以留一个联系方式吗？想请教一下你项目运行的问题
